static/app/components/searchSyntax/parser.tsx

```diff
@@ -773,6 +774,14 @@ export class TokenConverter {
     return !this.config.flattenParenGroups;
   };
 
+  /**
+   * When enableContainsCheck is enabled, the parser will check for the contains
+   * flag in text values. i.e. `*value*`
+   */
+  enableContainsCheck = (): boolean => {
```

Comment:
```suggestion
  containsCheckIsEnabled = (): boolean => {
```

Right now this sounds like a verb

Reply:
Going to combine this with @malwilley's suggestion expanding beyond just `contains` for some other wildcard checks that'll be adding in shortly.